cmd/juju/application/removeunit.go

```diff
@@ -281,7 +324,47 @@ func (c *removeUnitCommand) removeUnits(ctx *cmd.Context, client RemoveApplicati
 	return nil
 }
 
+func (c *removeUnitCommand) logResult(
+	ctx *cmd.Context,
+	name string,
+	result params.DestroyUnitResult,
+	errorOnly bool,
+) error {
+	if result.Error != nil {
+		err := errors.Annotatef(result.Error, "removing unit %s failed", name)
+		_, _ = fmt.Fprintf(ctx.Stderr, "%s\n", err)
```

Comment:
Not sure I agree. Readability is pretty good as it is. This will be the difference between:
```
WARNING! This command:
will remove unit ubuntu/0
will remove unit ubuntu/1
- will detach storage data/0

Continue? [y/N]
```
and 
```
WARNING! This command:
        will remove unit ubuntu/0
        will remove unit ubuntu/1
        - will detach storage data/0

Continue [y/N]
```
Prefixing with a tab seems un necessary to me, but that's just me